influxql/point.go

```diff
@@ -152,3 +153,13 @@ func encodeTags(m map[string]string) []byte {
 	}
 	return b
 }
+
+// ParseTags returns an instance of Tags for a comma-delimited list of key/values.
+func ParseTags(s string) Tags {
```

Comment:
This is currently just for testing to parse some simple cases. I can move it to the test package.